### PR TITLE
[web][SIMS #161]Student Application Privacy Updates - review

### PR DESCRIPTION
**Pull request by henryoforeh-aot**
I removed the columns in the **Personal Information** tab I missed in the previous PR.
personalinformation

### CODE REVIEW

File: sources/packages/forms/src/form-definitions/sfaa2023-24.json [hunk 1]
```diff
@@ -16506,12 +16305,13 @@
...
                     },
                     "errorLabel": "",
+                    "errors": "",
                     "key": "numberOfParents",
                     "tags": [],
                     "properties": {},
                     "conditional": {
                         "show": true,
-                        "when": "isYourSpouseACanadianCitizen",
+                        "when": "doYourParentSStepParentSponsorLegalGuardianHaveAValidSinNumber",
```
Comment (guru-aot) [inline sources/packages/forms/src/form-definitions/sfaa2023-24.json:16314]: this is added by me as part of my change

File: sources/packages/forms/src/form-definitions/sfaa2022-23.json [hunk 2]
```diff
@@ -16501,12 +16300,13 @@
...
                     },
                     "errorLabel": "",
+                    "errors": "",
                     "key": "numberOfParents",
                     "tags": [],
                     "properties": {},
                     "conditional": {
                         "show": true,
-                        "when": "isYourSpouseACanadianCitizen",
+                        "when": "doYourParentSStepParentSponsorLegalGuardianHaveAValidSinNumber",
```
Comment (guru-aot) [inline sources/packages/forms/src/form-definitions/sfaa2022-23.json:16309]: this is added by me as part of my change

File: sources/packages/forms/src/form-definitions/sfaa2021-22.json [hunk 3]
```diff
@@ -16457,70 +16266,69 @@
...
+                    "errorLabel": "",
+                    "errors": "",
                     "key": "numberOfParents",
+                    "tags": [],
+                    "properties": {},
                     "conditional": {
                         "show": true,
-                        "when": "isYourSpouseACanadianCitizen",
-                        "eq": "yes"
+                        "when": "doYourParentSStepParentSponsorLegalGuardianHaveAValidSinNumber",
```
Comment (guru-aot) [inline sources/packages/forms/src/form-definitions/sfaa2021-22.json:16298]: This is a bug so I added this as part of my change

File: sources/packages/forms/src/form-definitions/sfaa2022-23.json [hunk 4]
```diff
@@ -14044,7 +13841,7 @@
                             "id": "e9e67m6"
                         },
                         {
-                            "label": "I have been authorized by my partner to submit their financial information including their net income (previous tax year's return's Line 15000 information)",
+                            "label": "I have been authorized by my partner to submit their financial information including their net income (previous tax year's return's  Line 15000 information)",
```
Comment (guru-aot) [inline sources/packages/forms/src/form-definitions/sfaa2022-23.json:13844]: Not sure why this is happening again, there is an extra space after return's

File: sources/packages/forms/src/form-definitions/sfaa2023-24.json [hunk 5]
```diff
@@ -14049,7 +13846,7 @@
                             "id": "evpffp"
                         },
                         {
-                            "label": "I have been authorized by my partner to submit their financial information including their net income (previous tax year's return's Line 15000 information)",
+                            "label": "I have been authorized by my partner to submit their financial information including their net income (previous tax year's return's  Line 15000 information)",
```
Comment (guru-aot) [inline sources/packages/forms/src/form-definitions/sfaa2023-24.json:13849]: Not sure why this is happening again, there is an extra space after return's

File: sources/packages/forms/src/form-definitions/sfaa2023-24.json (hunk 1 above)
Comment (ann-aot) [reply sources/packages/forms/src/form-definitions/sfaa2023-24.json:16314]: I didn't @guru-aot . what is the bug here? name of the variable?

**Review by ann-aot — APPROVED** [review]
LGTM

**Review by abschwenker — APPROVED** [review]

**Review by andrewsignori-aot — APPROVED** [review]
looks good 👍

File: sources/packages/forms/src/form-definitions/sfaa2023-24.json (hunk 1 above)
Comment (guru-aot) [reply sources/packages/forms/src/form-definitions/sfaa2023-24.json:16314]: no the number of parents question is associated with the different yes or no, so changed it.

**Review by guru-aot — APPROVED** [review]
LGTM

**Review by dheepak-aot — APPROVED** [review]
LGTM.

**Pull request merged**